projects/storefrontlib/src/shared/components/truncate-text-with-popover/truncate-text-with-popover.module.ts

```diff
@@ -0,0 +1,14 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { I18nModule } from '@spartacus/core';
+import { PopoverModule } from '../popover/popover.module';
+import { TruncateTextWithPopoverComponent } from './truncate-text-with-popover.component';
+import { TruncatePipe } from './truncate.pipe';
+
+@NgModule({
+  imports: [CommonModule, I18nModule, PopoverModule],
```

Comment:
I don't think that you need `I18nModule` here

Reply:
button content `{{ 'common.more' | cxTranslate }}`